src/Microsoft.DotNet.Build.Tasks/GetPackageVersion.cs

```diff
@@ -27,42 +26,19 @@ public override bool Execute()
         {
             Log.LogMessage("Starting GetPackageVersion");
 
-            using (var repository = new Repository(RepositoryRoot))
```

Comment:
We can but I suspect we're going to need it in the future, or if we want to use it for committing and pushing the build number changes.